tests/helpers/cucumber-hooks.conf.js

```diff
@@ -7,9 +7,10 @@ exports.config = Object.assign({}, config, {
         await browser.pause(30)
         browser.Cucumber_Test = 0
     },
-    beforeScenario: () => {
+    beforeScenario: (world) => {
```

Comment:
`world` should be the 5th parameter, no?

Reply:
I would think so but was modeling it on before/afterStep which also only pass the newly added parameters:

`beforeStep: async function ({ step }, world) {
        await browser.pause(20)
        browser.Cucumber_Test += 2
        browser.Cucumber_CurrentStepText = step.step.text
        browser.Cucumber_CurrentWorld = world
    },
    afterStep: function ({ step }, world) {
        browser.pause(25)
        if (
            browser.Cucumber_CurrentStepText !== step.step.text ||
            browser.Cucumber_CurrentWorld !== world
        ) {
            throw new Error("step data doesn't match")
        }
        browser.Cucumber_Test = 1
    },`

Reply:
Managed to link @wdio/cucumber-framework but it doesn't seem to be working at all.  (In fact, can't even get context to console.log in before/afterStep where it is supposed to already be supported).  Will close PR for now.